framework/xds_k8s_testcase.py

```diff
@@ -411,6 +411,55 @@ def assertSuccessfulRpcs(
         )
         return lb_stats
 
+    # Return number of servers using subchannels
```

Comment:
I don't think we can derive it from the number of subchannels. One of the reasons: b/284497334. I'll think if this will make sense in a general case - after this issue is fixed.